File: Backup Assistant/Modals/About.vb
Public Class About

    ''' <summary>
    ''' Creates a new About form.
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
        InitializeComponent()

        lbl_ApplicationName.Text = Application.ProductName
        lbl_Version.Text = "Version " & Application.ProductVersion.Substring(0, Application.ProductVersion.Length - 2)
    End Sub

    Private Sub btn_OK_Click(sender As System.Object, e As System.EventArgs) Handles btn_OK.Click
        Me.Close()
    End Sub
End Class

File: src/Modals/About.cs
using System;
using System.Windows.Forms;

namespace BackupAssistant.Modals
{
    public partial class About : Form
    {
        public About()
        {
            InitializeComponent();

            this.About_Label_ApplicationName.Text = Application.ProductName;
            this.About_Label_Version.Text = "Version " + Application.ProductVersion;
        }

        private void About_Button_OK_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}
